src/components/tabs/TabsTriggerList.tsx

```diff
@@ -0,0 +1,166 @@
+  css,
+  children,
+  theme,
+  enableTabScrolling,
+  ...rest
+}) => {
+  const triggerListRef = useRef<HTMLDivElement>(null)
+  const [showLeftScroller, setShowLeftScroller] = useState<boolean>(false)
+  const [showRightScroller, setShowRightScroller] = useState<boolean>(false)
+
+  const scrollTriggerListTo = useCallback((direction: 'left' | 'right') => {
+    const triggerList = triggerListRef.current
+    if (triggerList) {
+      const { scrollWidth, scrollLeft } = triggerList
+      const scrollAmount = Math.round(scrollWidth / 4) // 25% of whole scroll width
+      let left = scrollLeft
+      if (direction === 'right') {
+        const newScrollAmount = triggerList.scrollLeft + scrollAmount
+        left = newScrollAmount <= scrollWidth ? newScrollAmount : scrollLeft
+      } else {
+        const newScrollAmount = scrollLeft - scrollAmount
+        left = newScrollAmount > 0 ? newScrollAmount : 0
+      }
+
+      triggerList.scroll({
+        left,
+        behavior: 'smooth'
+      })
+
+      // Relying on setTimeout since scroll does not have a callback / doesn't return a promise :(
+      setTimeout(() => {
+        const { scrollWidth, scrollLeft, offsetWidth } = triggerList
+        if (scrollLeft === 0) {
+          setShowLeftScroller(false)
+          setShowRightScroller(true)
+        } else if (scrollLeft + offsetWidth === scrollWidth) {
+          setShowRightScroller(false)
+          setShowLeftScroller(true)
+        } else {
+          setShowLeftScroller(true)
+          setShowRightScroller(true)
+        }
+      }, 300)
+    }
+  }, [])
+
+  useEffect(() => {
+    const triggerList = triggerListRef.current
+    if (triggerList) {
+      const { offsetWidth, scrollWidth } = triggerList
+
+      const shouldShowScroller = scrollWidth > offsetWidth
+      if (shouldShowScroller) {
+        setShowRightScroller(true)
+      }
+    }
+  }, [])
+
+  const showScroller =
+    showLeftScroller || showRightScroller || enableTabScrolling
+
+  if (showScroller) {
+    return (
+      <Flex css={{ position: 'relative' }}>
+        {showLeftScroller && (
+          <StyledChevronIcon
+            size="lg"
+            role="scrollbar"
+            label="scroll-left"
+            theme={theme}
+            onClick={() => scrollTriggerListTo('left')}
+            css={{ left: 0 }}
+          >
+            <Icon is={ChevronLeft} size="lg" />
+          </StyledChevronIcon>
+        )}
+        <StyledTriggerList
+          {...rest}
+          ref={triggerListRef}
+          theme={theme}
+          css={{
+            ...css,
+            width: '100%',
+            overflowX: 'auto',
+            '&::-webkit-scrollbar': { display: 'none' }
+          }}
+        >
+          {passPropsToChildren(children, { theme }, [TabTrigger])}
+        </StyledTriggerList>
+        {showRightScroller && (
+          <StyledChevronIcon
+            size="lg"
+            role="scrollbar"
+            label="scroll-right"
```

Comment:
Can we treat this label as human readable text? e.g.

```suggestion
            label="Scroll right"
```